src/javascripts/components/search.js

```diff
@@ -36,6 +50,7 @@ Search.prototype.fetchSearchIndex = function (indexUrl, callback) {
         callback(json)
       } else {
         statusMessage = 'Failed to load the search index'
+        // Log to analytics?
```

Comment:
Is this an intentional comment?

Reply:
Not sure, it's been there since the file was added https://github.com/alphagov/govuk-design-system/blob/841f04b119e48f43773d205672a960a56add0b01/src/javascripts/components/search.js#L34

Reply:
Hm, probably best to remove this line for now then I think?